components/Card/CardTech/CardTech.js

```diff
@@ -0,0 +1,78 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import React from 'react';
+import { detailData, techData } from '../../data/techData';
+
+export default function CardTech({ title, titleLink }) {
```

Comment:
The CardTech component can be renamed to only Tech component Because the CardTech component is inside the card folder.